core/src/main/scala/org/broadinstitute/dsde/rawls/config/ServicePerimeterServiceConfig.scala

```diff
@@ -0,0 +1,31 @@
+package org.broadinstitute.dsde.rawls.config
+
+import com.typesafe.config.Config
+import org.broadinstitute.dsde.rawls.model.{GoogleProjectNumber, ServicePerimeterName}
+
+import scala.collection.JavaConverters._
+import scala.concurrent.duration._
+import scala.language.postfixOps
+
+
+final case class ServicePerimeterServiceConfig(staticProjectsInPerimeters: Map[ServicePerimeterName, Seq[GoogleProjectNumber]] = Map.empty,
+                                               pollInterval: FiniteDuration = 5 seconds,
```

Comment:
I think 5s is too slow for production. How about 1s?

Reply:
I can change it to 1s in the firecloud-develop PR. I'm going to remove the default specified here because I think it's a little misleading to have this when it's really specified in config